Sources/UILocalNotification+Sweet.swift

```diff
@@ -33,8 +46,21 @@
             return filteredNotifications.first
         }
 
-        public static func delete(_ id: String) {
-            if let notification = find(id) {
+
+        /// Cancels a local notification.
+        ///
+        /// - Parameter id: The locala notification id.
+        public static func cancel(using id: String) {
```

Comment:
So makes sense right? 
schedule *with* new ID
find *for* existing ID
cancel *for* existing ID